docs/semgrep-code/supported-languages-python.md

```diff
@@ -16,15 +16,15 @@ Semgrep’s Python coverage leverages framework-specific analysis capabilities t
 :::
 
 ## Analysis Capabilities for Python
+* Framework-specific control flow analysis 
 * Inter-file analysis (cross-file)
 * Inter-procedural analysis (cross-function)
-* Framework-specific control flow analysis 
 
 ## Coverage 
 _Our coverage ethos: comprehensive and accurate detection of common, OWASP Top 10 issues in source code_
 
 
-Semgrep supports **over 100** common Python frameworks and libraries, including the 3 most popular frameworks: 
+Semgrep Code has native support for the most common web frameworks: 
```

Comment:
@chushili1 What does "native support" mean?

Reply:
Native support means that in addition to rules, the engine itself has built out capabilities for handling implicit dataflows in these frameworks (Django, Flask, FastAPI)

Reply:
Note for me to help with the rephrasing: 
![image](https://github.com/user-attachments/assets/e82786e1-f67a-4238-8f3a-1fb2c69163bf)

Reply:
```suggestion
In addition to rules, the Semgrep engine itself can analyze code and implicit dataflows in the context of the following supported frameworks:
```
Or something similar!